### ISSUE TITLE
CREATE INDEX response string returns "CREATE TABLE"

### ISSUE THREAD

**Issue by fadgan** [cmu-db/peloton#787]
Index creation output tells us as if a table has been created although catalog confirms that the correct actions have been performed under the commands:
```
postgresql=# create table foo3 (id integer);
CREATE TABLE
postgresql=# insert into foo3 values (1);
INSERT 0 1
postgresql=# create index ind on foo3(id);
CREATE TABLE
postgresql=# select * from pg_catalog.pg_index where index_name = 'ind';
 index_oid | index_name | table_oid | index_type | index_constraint | unique_keys | indexed_attributes 
-----------+------------+-----------+------------+------------------+-------------+--------------------
  50331759 | ind        |  33554538 |          1 |                1 | false       | 0 
(1 row)

```

**Comment by apavlo** [comment]
Good find. We will fix.

**Comment by apavlo** [comment]
@sivaprasadsudhir Can you take care of this? The function to modify is here:

https://github.com/cmu-db/peloton/blob/master/src/network/protocol_handler.cpp#L278

You should coordinate with @ChTimTsubasa about this change so that you don't foul up overhaul of the protocol handler code.

**Comment by apavlo** [comment]
Fixed in #805
